docs/setup/developer.md

```diff
@@ -2,6 +2,18 @@
 
 ## Setting up `Python` Virtual Environment (VE)
 
+### Using `uv` (stringly recommended)
+Once you have installed `uv`, in the root of your source tree you can:
+
+Create the venv (one-off task)
+```
+uv venv
+```
```

Comment:
``````suggestion
Once you have installed `uv`, in the root of your source tree you can:

Install the required Python version and create the venv (one-off task)
```
uv python install 3.11
uv venv --python 3.11
```
``````